app/javascript/src/hyacinth_v1/components/digital_objects/rights/RightsEdit.js

```diff
@@ -33,14 +33,14 @@ function RightsEdit(props) {
     return (<GraphQLErrors errors={digitalObjectError || rightsFieldsError} />);
   }
 
-  const { digitalObject, digitalObject: { digitalObjectType } } = digitalObjectData;
+  const { digitalObject, digitalObject: { digitalObjectType }, digitalObject: { title } } = digitalObjectData;
```

Comment:
```suggestion
  const { digitalObject, digitalObject: { digitalObjectType, title } } = digitalObjectData;
```
I forgot -- does this work?

Reply:
I'm pretty sure nested destructures work, yes!

Reply:
Done!